src/test/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepositoryTest.java

```diff
@@ -421,6 +421,22 @@ public void addFutureBuildTasks_doesnt_schedule_build_if_postPullRequestComment_
     assertThat(Jenkins.getInstance().getQueue().getItems(), is(emptyArray()));
   }
 
+  @Test
+  public void pollRepository_schedules_build_for_open_pull_request() throws Exception {
+    List<StashPullRequestComment> comments = Collections.emptyList();
+    StashPullRequestComment response = new StashPullRequestComment();
+    response.setCommentId(1);
+
+    when(trigger.getStashHost()).thenReturn("StashHost");
+    when(stashApiClient.getPullRequests()).thenReturn(pullRequestList);
+    when(stashApiClient.getPullRequestComments(any(), any(), any())).thenReturn(comments);
```

Comment:
minor, I would just use `emptyList()` inline here, you don't have to jump back to the top to see what it is then

Reply:
Done.